root/static/scripts/common/components/Relationship.js

```diff
@@ -0,0 +1,178 @@
+/*
+ * @flow
+ * Copyright (C) 2020 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import * as React from 'react';
+
+import linkedEntities from '../linkedEntities';
+import {commaOnlyListText} from '../i18n/commaOnlyList';
+import {bracketedText} from '../utility/bracketed';
+import formatDate from '../utility/formatDate';
+import isDateEmpty from '../utility/isDateEmpty';
+import {interpolate, getExtraAttributes} from '../../edit/utility/linkPhrase';
+import type {LinkPhraseProp} from '../../edit/utility/linkPhrase';
+
+import DescriptiveLink from './DescriptiveLink';
+
+type Props = {
+  +phraseField?: LinkPhraseProp,
+  +relationship: RelationshipT,
+};
+
+const RelationshipDates = ({relationship}: {relationship: RelationshipT}) => {
```

Comment:
This looks like it re-implements root/utility/relationshipDateText.js - can we use that instead?